presto-native-execution/presto_cpp/presto_protocol/connector/clp/presto_protocol_clp.proto

```
@@ -0,0 +1,37 @@
+package presto_protocol;
+
+
+message ClpColumnHandle {
+    String columnName = 1;
+    String originalColumnName = 2;
+    Type columnType = 3;
+    bool nullable = 4;
+}
+
+enum ArchiveType {
+    UNKNOWN = 1;
```

Buf (1.47.2)

enum presto_protocol.ArchiveType: proto3 requires that first value of enum have numeric value zero

(COMPILE)